websites/R/R-a-d.io/presence.ts

```diff
@@ -0,0 +1,94 @@
+const presence = new Presence({
+		clientId: "1073136952905318401",
+	}),
+	/**
+	 * Returns current player status
+	 * @returns {string} either "Stopped", "Playing", or "Connecting..."
+	 */
+	playStatus = (): string => {
+		let status: string;
+		const playButtonText: string =
+			document.querySelector("#stream-play").textContent;
+		if (playButtonText === "Play Stream") status = "Stopped";
+		else if (playButtonText === "Stop Stream") status = "Playing";
+		else status = playButtonText;
+		return status;
+	},
+	/**
+	 * Returns current song playing
+	 * @returns {string} Current song, usually in "{artist} - {title}" format
+	 */
```

Comment:
jsdoc over complicates the code base for something this small

Reply:
Should I just remove the return value from this JSDoc or just revert it all back to regular comment?

Reply:
What's the "regular comment"?

Reply:
`//`

Reply:
For now I think I'll just remove the `@return`. Let me know if it needs more adjustments!